.github/workflows/main.yml

```
@@ -0,0 +1,61 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        python-version: [3.7, 3.8]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
```

```suggestion
      - name: Set up Python
```
Saves 29 characters.

---

Always good to save some virtual ink